ui2/src/pages/manage/Manage.tsx

```diff
@@ -1,19 +1,46 @@
-import React from 'react';
-import { NavLink } from 'react-router-dom';
-import { Menu } from 'semantic-ui-react';
+import React, { useEffect, useState } from 'react';
+import { NavLink, useHistory, useLocation } from 'react-router-dom';
+import { Menu, Form } from 'semantic-ui-react';
 
 const Manage: React.FC = ({ children }) => {
+  const history = useHistory();
+
+  let path = useLocation().pathname;
+
+  const menuItems = [
+    { text: 'Custody', value: '/app/manage/custody' },
+    { text: 'Distributions', value: '/app/manage/distributions' },
+    { text: 'Instruments', value: '/app/manage/instruments' },
+    { text: 'Issuances', value: '/app/manage/issuance' },
+    { text: 'Listings', value: '/app/manage/listings' },
+    { text: 'Trading', value: '/app/manage/trading' },
+  ];
+
+  const [currentMenuItem, setCurrentMenuItem] = useState<{ text: string; value: string }>();
+  useEffect(() => {
+    const item = menuItems.find(i => path === i.value);
+    if (item) {
+      console.log(item);
+
+      setCurrentMenuItem(item);
+    }
+  }, [path]);
+
+  console.log(currentMenuItem?.text);
+
   return (
     <div className="manage">
       <Menu className="manage-menu" pointing secondary>
-        <Menu.Item as={NavLink} name="custody" to="/app/manage/custody" />
-        <Menu.Item as={NavLink} name="distributions" to="/app/manage/distributions" />
-        <Menu.Item as={NavLink} name="instruments" to="/app/manage/instruments" />
-        <Menu.Item as={NavLink} name="issuances" to="/app/manage/issuance" />
-        <Menu.Item as={NavLink} name="listings" to="/app/manage/listings" />
-        <Menu.Item as={NavLink} name="trading" to="/app/manage/trading" />
+        {menuItems.map(item => (
+          <Menu.Item as={NavLink} name={item.text} to={item.value} />
+        ))}
       </Menu>
-
+      <Form.Select
+        placeholder="Select..."
+        options={menuItems}
+        value={currentMenuItem?.value}
+        onChange={(_, data: any) => history.push(data.value)}
+      />
```

Comment:
What is this form select for?

Reply:
I checked out the branch, and it seems like its hidden but also not necessary (nav links still worked when removing it)

Reply:
it is the menu when the page is responsive, try shrinking the page - it will show up!

Reply:
<img width="1111" alt="Screen Shot 2021-04-13 at 7 36 43 PM" src="https://user-images.githubusercontent.com/40438835/115050659-fa3e9880-9ea9-11eb-8b82-2d2f9d4884fe.png">

Reply:
<img width="1032" alt="Screen Shot 2021-04-16 at 11 51 10 AM" src="https://user-images.githubusercontent.com/40438835/115050728-0c203b80-9eaa-11eb-8b49-72020fdd3304.png">

Reply:
of course! 😁  thanks, good work